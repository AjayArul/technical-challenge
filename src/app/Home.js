import React from 'react'

const Home = () => {
    return (
        <div className='container-fluid welcome'>
            <span> welcome </span>
        </div>
    )
}

export default Home
