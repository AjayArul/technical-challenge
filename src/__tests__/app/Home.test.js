import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Home from './../../app/Home';

describe('Home test <Home />', () => {
  let _wrapper
  beforeEach(() => {
    _wrapper = shallow(<Home />)
  })
  it('Should Defined and render <Home /> component', () => {
      expect(_wrapper).toBeDefined()
      expect(_wrapper).toMatchSnapshot()
  });
});
