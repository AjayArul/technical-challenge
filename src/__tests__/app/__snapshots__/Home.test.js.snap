// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home test <Home /> Should Defined and render <Home /> component 1`] = `ShallowWrapper {}`;
